packages/neth/src/lib/neth.ts

```diff
@@ -150,12 +163,14 @@ const Neth: WalletBehaviourFactory<InjectedWallet> = async ({
 
 export function setupNeth({
   useModalCover = false,
+  bundle: _bundle = true,
```

Comment:
You can remove _bundle, and just define it with as `bundle = true` , and then remove line 173.

Reply:
I don't think that will work. The goal is to store bundle arg value in the bundle var local to the module.